app/Http/Controllers/ViewproductController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class ViewproductController extends Controller
```

Comment:
```suggestion
class ProductController extends Controller
```
Trong Controller này em sẽ thực hiện các action liên quan đến product, ko chỉ là view. chú ý phần này